align_system/algorithms/outlines_regression_adm_comparative.py

```diff
@@ -280,11 +281,26 @@ def sample_kdma_score_predictions(self,
         return predictions
 
 
+    # TODO - this could be improved
+    def get_selected_choice_reasoning(self, selected_choice, predicted_kdma_values, target_kdmas):
+        # If outcomes were predicted, add first one to reasoning
+        if predicted_kdma_values[selected_choice]['predicted_outcome'][0] is not None:
+            reasoning = 'The predicted outcome for choice ' + selected_choice + ' was: '
+            reasoning += predicted_kdma_values[selected_choice]['predicted_outcome'][0]
+        else:
+            reasoning = ''
+        # Add predicted KDMA scores to reasoning
+        for target_kdma in target_kdmas:
+            reasoning += ' The predicted scores for ' + target_kdma['name'] + ' were ' + \
+                            str(predicted_kdma_values[selected_choice][target_kdma['kdma']]['score']) + '.'
```

Comment:
Nit: f-strings might be more readable here

Reply:
Fixed https://github.com/ITM-Kitware/align-system/pull/82/commits/dfadd402dc323ce4116b00fb5f99ae0900841f7b